### ISSUE TITLE
blogdown:::serve_site() Error: Failed to run the command in the background

### ISSUE THREAD

**Issue by Yangxinyee** [gohugoio/hugo#11436]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
0.118.2
When I want to view a web page in the viewer：
> blogdown:::serve_site()
Error: Failed to run the command in the background: "C:/Users/24372/AppData/Roaming/Hugo/0.118.2/hugo.exe" "server" "--bind" "127.0.0.1" "-p" "4321" "--themesDir" "themes" "--theme" "anatole" "-D" "-F" "--navigateToChanged"

</pre>

**Comment by bep** [comment]
You need to provide more information.

**Comment by Yangxinyee** [comment]
I used RStudio, blogdown and Hugo. All of them are latest version.
**Frist**, I downloaded blogdown package in RStudio. No error generated.
**Second**, I tried command:
`blogdown::new_site(theme = 'yihui/hugo-ivy')`
and
`blogdown::new_site(theme = 'lxndrblz/anatole')'
Still, no error generated. 
**However**, in console, it showed: 
`Want to serve and preview the site now? (y/n)`
I typed "y". Then here came the error: 
`Error: Failed to run the command in the background: "C:/Users/xxxx/AppData/Roaming/Hugo/0.118.2/hugo.exe"`

I tried redownloading hugo package and add the directory ("C:/Users/xxxx/AppData/Roaming/Hugo/0.118.2/hugo.exe") to the PATH environment variable as Hugo's doc says.
And I tried reinstalling all the tools, such as Git, Go, R, RStudio and Hugo. All of them are latest version.
The problem is still unresolved...
